### PR TITLE
Preview events

### PR DESCRIPTION
**Pull request by hannahcheren**
https://path-cc.io/web-preview/preview-events/events/OSG-User-School-2023/

### CODE REVIEW

File: _events/OSG-User-School-2023.md [hunk 1]
```diff
@@ -18,14 +18,17 @@ header_image: "/images/Madison_Skyline.jpg"
 
 The OSG User School will occur on August 7–11, where you will learn to use high-throughput computing (HTC) systems to run large-scale computing applications at your campus or using the national-scale OSG.
```
Comment (osg-cat) [inline _events/OSG-User-School-2023.md:19]: As the first content line of this page, could it be made more interesting and appealing?

File: _events/OSG-User-School-2023.md [hunk 2]
```diff
@@ -18,14 +18,17 @@ header_image: "/images/Madison_Skyline.jpg"
...
 
-The School is ideal for researchers in any research area where large-scale computing is a vital part of the research process, anyone, especially students and staff, who supports researchers and are current or potential users of high-throughput computing, and instructors who teach future researchers and see value in integrating high-throughput computing into their curriculum.
-
 Applications are now open. The deadline for applications is Monday, 17 April 2023. Submit your application soon to ensure consideration!
```
Comment (osg-cat) [inline _events/OSG-User-School-2023.md:21]: I don’t think that this information belongs here.

File: _events/OSG-User-School-2023.md [hunk 3]
```diff
@@ -18,14 +18,17 @@ header_image: "/images/Madison_Skyline.jpg"
...
 
 To apply, complete the [online application form](https://uwmadison.co1.qualtrics.com/jfe/form/SV_9shkUL5ZDSghKvk) and have someone email a letter of recommendation for you to [user-school@osg-htc.org](mailto:user-school@osg-htc.org).
```
Comment (osg-cat) [inline _events/OSG-User-School-2023.md:23]: Remove.

File: _events/OSG-User-School-2023.md [hunk 4]
```diff
@@ -18,14 +18,17 @@ header_image: "/images/Madison_Skyline.jpg"
...
 
 For more information about the School and to stay up-to-date with any announcements, please visit the [OSG User School 2023 site](https://osg-htc.org/user-school-2023/).
```
Comment (osg-cat) [inline _events/OSG-User-School-2023.md:25]: Remove.

File: _events/OSG-User-School-2023.md [hunk 5]
```diff
@@ -18,14 +18,17 @@ header_image: "/images/Madison_Skyline.jpg"
...
 
+To learn more about the event, check out these articles written about the OSG User School 2022:
```
Comment (osg-cat) [inline _events/OSG-User-School-2023.md:27]: Instead of just a list of links, can each one have a sentence or so that might actually draw people in?

**Review by osg-cat — CHANGES_REQUESTED** [review]
Comments:

- Shorten the title so it fits better: title: "OSG User School 2023, August 7–11" => "OSG User School 2023, Aug. 7–11"

- In the metadata, change the link: link: "https://opensciencegrid.org/user-school-2023/#applications" => "https://osg-htc.org/user-school-2023/". I just want this (events) page to link to the overall home page, nothing extra.

- Please find an image from a past in-person School, rather than something generic (and yes, I know what’s on the School’s home page). If Janet does not have images to choose from, we can find a time for you to look through my collection.

- There’s just too much repetition of ideas and words here. We need appealing content!

File: _events/OSG-User-School-2023.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+header_image: "/images/osg-user-school-2023-event.jpg"
+---
+
+{% capture main %}
+
+The OSG User School provides researchers the opportunity to learn how to use high-throughput computing (HTC) systems to run large-scale computing applications at their campus or using the national-scale OSG consortium. Please join us and learn how you can use HTC methods and services to handle large-scale computing applications to advance your research!
+
+To learn more about the event, check out these articles written about the OSG User School 2022:
+- Read about last year's [OSG User School student lightning talks](https://osg-htc.org/spotlights/Lightning-Talks.html), where researchers showcased their work and future plans to implement HTC.
+- This article spotlights [studens from the NIAID/ACE students who attended this year’s OSG User School 2022](https://osg-htc.org/spotlights/NIAID-ACE-students-attend-OSG-User-School.html). Learn more about these students and their work and the ongoing collaboration between the OSG and ACE/NIAID teams.
```
Comment (osg-cat) [inline _events/OSG-User-School-2023.md:23]: “students”

Comment (osg-cat) [inline _events/OSG-User-School-2023.md:23]: Need to spell out NIAID and ACE on first use.

File: _events/OSG-User-School-2023.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+end_date: 2023-08-11
+location: "University of Wisconsin-Madison"
+link: "https://osg-htc.org/user-school-2023/"
+image: "/images/osg-user-school-2023-event.jpg"
+header_image: "/images/osg-user-school-2023-event.jpg"
+---
+
+{% capture main %}
+
+The OSG User School provides researchers the opportunity to learn how to use high-throughput computing (HTC) systems to run large-scale computing applications at their campus or using the national-scale OSG consortium. Please join us and learn how you can use HTC methods and services to handle large-scale computing applications to advance your research!
```
Comment (osg-cat) [inline _events/OSG-User-School-2023.md:19]: “OSG Consortium” [capital C]

Comment (osg-cat) [inline _events/OSG-User-School-2023.md:19]: Replace paragraph with:

“
Has your research computing outgrown your available capacity? How could access to **lots** more computing transform your research or that of others? If you have research workloads that can be broken into many independent, parallel computing tasks, we can help.

The OSG User School provides researchers the opportunity to learn how to use high-throughput computing (HTC) systems to run large-scale computing applications at their campus or using the national-scale OSG consortium.
”

[Or feel free to edit, but the existing paragraph has two sentences that say almost the same thing. I stole the other text from our announcement email.]

File: _events/OSG-User-School-2023.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+To learn more about the event, check out these articles written about the OSG User School 2022:
+- Read about last year's [OSG User School student lightning talks](https://osg-htc.org/spotlights/Lightning-Talks.html), where researchers showcased their work and future plans to implement HTC.
+- This article spotlights [studens from the NIAID/ACE students who attended this year’s OSG User School 2022](https://osg-htc.org/spotlights/NIAID-ACE-students-attend-OSG-User-School.html). Learn more about these students and their work and the ongoing collaboration between the OSG and ACE/NIAID teams.
+- To view last year's lesson plans and materials, check out the [OSG User School Concludes](https://osg-htc.org/spotlights/OSG-User-School-Concludes.html) article on the OSG website.
+
```
Comment (osg-cat) [inline _events/OSG-User-School-2023.md:25]: Add:

“
<span style="font-size: larger; color: #FF6600;">Apply by April 17 on [the OSG School website](https://osg-htc.org/user-school-2023/).</span>
”

[Yes, it repeats the stuff on the right, but I’m fine with this here.]

**Review by osg-cat — CHANGES_REQUESTED** [review]
I think this is getting pretty close. Please make the new changes, then carefully check over one more time, then go ahead and post.

**Comment by hannahcheren** [comment]
> I think this is getting pretty close. Please make the new changes, then carefully check over one more time, then go ahead and post.

**Pull request closed without merging**